tests/acceptance/CreateNewWordPressUserCest.php

```diff
@@ -15,18 +15,43 @@ public function _before() {
   }
 
   public function sendConfirmationEmail(\AcceptanceTester $i) {
-    $i->wantTo('Create a new wordpress user and check if the confirmation email is sent');
+    $i->wantTo('Create a new WP user and check if the confirmation email is sent and if user is subscribed properly');
     $this->settings->withConfirmationEmailEnabled();
     $this->settings->withSubscribeOnRegisterEnabled();
+    $secondListName = 'Newsletter mailing list';
+    $emailTitle = 'Confirm your subscription';
 
-    //create a wp user with wp role subscriber
-    $i->cli(['user', 'create', 'narwhal', 'standardtest@example.com', '--role=subscriber']);
-    $i->checkEmailWasReceived('Confirm your subscription');
-    $i->click(Locator::contains('span.subject', 'Confirm your subscription'));
+    // add additional list in settings
+    $i->login();
+    $i->amOnMailpoetPage('Settings');
+    $i->waitForText('Settings');
+    $i->selectOptionInSelect2($secondListName);
```

Comment:
There are several Select2 instances on the page, can you specify which one do you intend to click with a `data-automation-id` attribute?